DeepFlux for Skeletons in the Wild/README.md

```diff
@@ -0,0 +1,99 @@
+# DeepFlux for Skeletons in the Wild
+
+[The DagsHub Repo](https://dagshub.com/Bharat-mtr/DeepFlux)
+
+Computing object skeletons in natural images is challenging, owing to large variations in object appearance and scale, and the complexity of handling background clutter. Many recent methods frame object skeleton detection as a binary pixel classification problem, which is similar in spirit to learning-based edge detection, as well as to semantic segmentation methods. In the present article, we depart from this strategy by training a CNN to predict a twodimensional vector field, which maps each scene point to a candidate skeleton pixel, in the spirit of flux-based skeletonization algorithms. This “image context flux” representation has two major advantages over previous approaches. First, it explicitly encodes the relative position of skeletal pixels to semantically meaningful entities, such as the image points in their spatial context, and hence also the implied object boundaries. Second, since the skeleton detection context is a region-based vector field, it is better able to cope with object parts of large width. We evaluate the proposed method on three benchmark datasets for skeleton detection and two for symmetry detection, achieving consistently superior performance over state-of-the-art methods.
+
+The code and trained models of:
+
+DeepFlux for Skeletons in the Wild, CVPR 2019 [[Paper]](http://openaccess.thecvf.com/content_CVPR_2019/papers/Wang_DeepFlux_for_Skeletons_in_the_Wild_CVPR_2019_paper.pdf)
+
+## Prerequisite
+
+- Caffe and VGG-16 pretrained model [[VGG_ILSVRC_16_layers.caffemodel]](http://www.robots.ox.ac.uk/~vgg/software/very_deep/caffe/VGG_ILSVRC_16_layers.caffemodel)
+
+- Datasets: [[SK-LARGE]](https://dagshub.com/Bharat-mtr/DeepFlux/src/master/data/SK-LARGE), [[SYM-PASCAL]](https://dagshub.com/Bharat-mtr/DeepFlux/src/master/data/SymPASCAL-by-KZ)
+
+- OpenCV 3.4.3 (C++ or Python, optional)
+
+- MATLAB
+
+## Usage
+
+#### 1. Install Caffe
+
+```bash
+
+cp Makefile.config.example Makefile.config
+# adjust Makefile.config (for example, enable python layer)
+make all -j16
+# make sure to include $CAFFE_ROOT/python to your PYTHONPATH.
+make pycaffe
+
+```
+
+Please refer to [Caffe Installation](http://caffe.berkeleyvision.org/install_apt.html) to ensure other dependencies.
+
+#### 2. Data and model preparation
+
+```bash
+
+# download datasets and pretrained model then
+mkdir data && mv [your_dataset_folder] data/
+mkdir models && mv [your_pretrained_model] models/
```

Comment:
The folder you added is called `model` instead of `models` like the instruction, so this might break training or inference, can you please change it?

Also, if this section has already been done, you can remove it from the instructions, since it isn't necessary any more

Reply:
Thanks for your suggestion. I have edited this part accordingly in the new changes.